### PR TITLE
Missing metadata in .nfo file

### PR DESCRIPTION
**Pull request by desepticon**
Couchpotato doesn't add all the metadata to the .nfo file for Kodi when generated. Reloading the movie manually (ignoring local) will add this missing metadata. Country and studio seem to be the only fields missing. Adding this here since I cant seem to add to issues section

### Steps to reproduce:
1. Download a movie through CP (although this can happen with manually scanned movies from other sources)
2. Have CP generate Kodi .nfo file (and artwork) when processing
3. In Kodi, I notice that some fields are missing such as Country, and Studio.
4. Reloading the movie in Kodi provides additional metadata

### Information:
Movie(s) I have this with: many
Quality of the movie being searched: Best
Providers I use: DogNzb
Version of CouchPotato: git:(CouchPotato:CouchPotatoServer develop) bb929bc8 (2019-01-29 09:45:55)
Running on: MacOS 10.13.6

### Logs:
n/a

### CODE REVIEW

File: couchpotato/core/notifications/plex/__init__.py [hunk 1]
```diff
@@ -23,6 +23,19 @@ def autoload():
                     'default': 'localhost',
                     'description': 'Hostname/IP, default localhost'
                 },
+                {
+                    'name': 'media_server_port',
+                    'label': 'Port',
+                    'default': '32400',
+                    'description': 'Connection tot he Media Server should use this port'
```
Comment (dynamiccookies) [inline couchpotato/core/notifications/plex/__init__.py]:
```suggestion
                    'description': 'Connection to the Media Server should use this port'
```

**Comment by dynamiccookies** [comment]
I am having the same issue as @desepticon regarding missing metadata, and the current manual process to fix it.

**Comment by desepticon** [comment]
@dynamiccookies Do you know why they closed the issues section by any chance? The forums have closed too. I would think couchpotato is dead but I get plenty of updates regularly.

**Comment by dynamiccookies** [comment]
@desepticon Unfortunately, I don't. I'm also curious if [Couchpotato](/CouchPotato/CouchPotatoServer) is on its way out. I really love it so, I truly hope it isn't, and as you mentioned, I've seen a number of updates coming from the repo since I'm signed up for any changes.

Can any moderators elaborate as to the future of Couchpotato?

**Comment by nydave69** [comment]
I am finding missing metadata also... 
The audio section is missing in the NFO and its also missing from the renamed files now.

file name settings
<thename> (<year>) (<imdb_id>) [<video>-<quality> <audio>-<audio_channels>] <cd>.<ext>

file renamed as;
Spotlight (2015) (tt1895587) [avc1-1080p-.mp4
It has the - that's the separator between audio - audio_channels
the audio is missing, just noticed it since everything seemed to be fine.. looks like it goes back a month or more. 
It also does this every once in a while, the movie was 1080p and labeled that way but CP did.
Return of the Jedi (1983) (tt0086190) [avc1-BR-Rip-.mp4
It marked the quality as BR-Rip for some reason...


ANyone know anyway to get the poster (folder) art to have the file name like this -fanart.
Spotlight (2015) (tt1895587) [avc1-1080p--fanart.jpg
rather then
folder.jpg

right now I'm using the TinyMediaManger to correct these movies....

hope we can get this fixed...

**Pull request merged**